packages/engine-server/src/drivers/file/storev2.ts

```diff
@@ -491,7 +492,19 @@ export class FileStorage implements DStore {
     // instantiate so we can use singleton later
     if (this.config.workspace.metadataStore === "sqlite") {
       // eslint-disable-next-line no-new
-      new SQLiteMetadataStore({ wsRoot: this.wsRoot, force: true });
+      const store = new SQLiteMetadataStore({
+        wsRoot: this.wsRoot,
+        force: true,
+      });
+
+      // TODO: stop sleeping after max timeout
```

Comment:
We should do this TODO now, else people will get stuck here.

Reply:
updated. not as worried here because this only gets invoked if users are opting in to using sqlite

Reply:
this change also has some other updates required to make the automated publishing pipeline work